python/cudf/cudf/core/dataframe.py

```diff
@@ -4379,7 +4379,7 @@ def query(self, expr, local_dict=None):
         1 2018-10-08
 
         .. pandas-compat::
-            **DataFrame.query**
+            :func:`pandas.DataFrame.query`
```

Comment:
We've been just putting the API name in bold and not cross-linking it with intersphinx. I generally like using intersphinx but we'll have to see how it renders. If this change is accepted, would you be willing to make all the other `.. pandas-compat::` sections match with an intersphinx reference in a second PR?

Reply:
> If this change is accepted, would you be willing to make all the other `.. pandas-compat::` sections match with an intersphinx reference in a second PR?

Should be ok.

Any idea why the docs build was skipped? https://github.com/rapidsai/cudf/actions/runs/8423601396/job/23067371447?pr=15383